src/script_pokemon_util.c

```diff
@@ -338,3 +338,65 @@ void ToggleGigantamaxFactor(struct ScriptContext *ctx)
+    u8 defEv      = VarGet(ScriptReadByte(ctx));
+    u8 speedEv    = VarGet(ScriptReadByte(ctx));
+    u8 spAtkEv    = VarGet(ScriptReadByte(ctx));
+    u8 spDefEv    = VarGet(ScriptReadByte(ctx));
+    u8 hpIv       = VarGet(ScriptReadByte(ctx));
+    u8 atkIv      = VarGet(ScriptReadByte(ctx));
+    u8 defIv      = VarGet(ScriptReadByte(ctx));
+    u8 speedIv    = VarGet(ScriptReadByte(ctx));
+    u8 spAtkIv    = VarGet(ScriptReadByte(ctx));
+    u8 spDefIv    = VarGet(ScriptReadByte(ctx));
+    u16 move1     = VarGet(ScriptReadHalfword(ctx));
+    u16 move2     = VarGet(ScriptReadHalfword(ctx));
+    u16 move3     = VarGet(ScriptReadHalfword(ctx));
+    u16 move4     = VarGet(ScriptReadHalfword(ctx));
+    bool8 isShiny = VarGet(ScriptReadByte(ctx));
+
+    u8 evs[NUM_STATS] = {hpEv, atkEv, defEv, speedEv, spAtkEv, spDefEv};
+    u8 ivs[NUM_STATS] = {hpIv, atkIv, defIv, speedIv, spAtkIv, spDefIv};
+    u16 moves[4] = {move1, move2, move3, move4};
+
+    gSpecialVar_Result = CreateCustomMon(species, level, item, ball, nature, abilityNum, gender, evs, ivs, moves, isShiny);
+}
+
+void Script_GetChosenMonOffensiveEV(void)
+{
+    ConvertIntToDecimalStringN(gStringVar1, GetMonData(&gPlayerParty[gSpecialVar_0x8004], MON_DATA_ATK_EV), STR_CONV_MODE_LEFT_ALIGN, 3);
+    ConvertIntToDecimalStringN(gStringVar2, GetMonData(&gPlayerParty[gSpecialVar_0x8004], MON_DATA_SPATK_EV), STR_CONV_MODE_LEFT_ALIGN, 3);
+    ConvertIntToDecimalStringN(gStringVar3, GetMonData(&gPlayerParty[gSpecialVar_0x8004], MON_DATA_SPEED_EV), STR_CONV_MODE_LEFT_ALIGN, 3);
+}
+
+void Script_GetChosenMonDefensiveEV(void)
+{
+    ConvertIntToDecimalStringN(gStringVar1, GetMonData(&gPlayerParty[gSpecialVar_0x8004], MON_DATA_HP_EV), STR_CONV_MODE_LEFT_ALIGN, 3);
+    ConvertIntToDecimalStringN(gStringVar2, GetMonData(&gPlayerParty[gSpecialVar_0x8004], MON_DATA_DEF_EV), STR_CONV_MODE_LEFT_ALIGN, 3);
+    ConvertIntToDecimalStringN(gStringVar3, GetMonData(&gPlayerParty[gSpecialVar_0x8004], MON_DATA_SPDEF_EV), STR_CONV_MODE_LEFT_ALIGN, 3);
+}
+
+void Script_GetChosenMonOffensiveIV(void)
+{
+    ConvertIntToDecimalStringN(gStringVar1, GetMonData(&gPlayerParty[gSpecialVar_0x8004], MON_DATA_ATK_IV), STR_CONV_MODE_LEFT_ALIGN, 3);
+    ConvertIntToDecimalStringN(gStringVar2, GetMonData(&gPlayerParty[gSpecialVar_0x8004], MON_DATA_SPATK_IV), STR_CONV_MODE_LEFT_ALIGN, 3);
+    ConvertIntToDecimalStringN(gStringVar3, GetMonData(&gPlayerParty[gSpecialVar_0x8004], MON_DATA_SPEED_IV), STR_CONV_MODE_LEFT_ALIGN, 3);
+}
+
+void Script_GetChosenMonDefensiveIV(void)
+{
+    ConvertIntToDecimalStringN(gStringVar1, GetMonData(&gPlayerParty[gSpecialVar_0x8004], MON_DATA_HP_IV), STR_CONV_MODE_LEFT_ALIGN, 3);
+    ConvertIntToDecimalStringN(gStringVar2, GetMonData(&gPlayerParty[gSpecialVar_0x8004], MON_DATA_DEF_IV), STR_CONV_MODE_LEFT_ALIGN, 3);
+    ConvertIntToDecimalStringN(gStringVar3, GetMonData(&gPlayerParty[gSpecialVar_0x8004], MON_DATA_SPDEF_IV), STR_CONV_MODE_LEFT_ALIGN, 3);
+}
```

Comment:
Is there a reason all of those are here and not in the debug menu?

Reply:
The debug menu can be toggled off and in that case those functions would only be limited to it even though they could be used by someone making a ROM Hack project for other purposes like an NPC or an item that reveals a Pokémon's IV.

I figured it'd be better to keep them in a more accessible place.

Reply:
If we disable the debug menu aren't the functions going to throw an error with modern?

Reply:
🤔 ...

Perhaps.

I'll check. If it does throw an error, I could define them in `data/specials.inc` so they can be used via `special`.

Reply:
Solved in the Discord server. No compilation errors, but I still defined the functions in `data/specials,inc` so they can be used through the `special` scripting command.